### DISCUSSION TITLE
Character, Stage & more Expansion system

### DISCUSSION THREAD

**Discussion by Bibidimah21** [Raytwo/ARCropolis#411] [Ideas] [answered]
The messeage say, Adding new fighters, articles, items, etc. is not currently possible, and likely won't be until someone like *you* does it.

Adding these things requires more intrusive modification to the game than ARCropolis will provide, and it's not on anyone's to-do list. If you are interested enough in pursuing adding new objects to the game like this, you are more than welcome to do the research and be the hero you are seeking, but it's not in the cards for the ARCropolis developers.

Character stacking is alternative, which doesn't involve adding new fighter kinds and new weapons, and you can see an example of that with Akuma in Championship Edition. Someday there might be an automated process for doing that, but that's also not available right now and you would need to look at Championship Edition for how to recreate it.

that's all i care to write someone can change it if they want but I have food to eeat

- blujay 2022
Adding new fighters, articles, items, etc. will possible coming to Arcropolis update like Paid DLC.

**Comment by Raytwo** [comment, marked as answer]
"Character stacking" is not character addition, it's just a bunch of very situational hacky garbage everybody praised as something revolutionary, I don't get what you're trying to say here.
